contrib/archlinux/PKGBUILD

```diff
@@ -14,12 +14,18 @@ install=
 source=("$pkgname"::'git+https://github.com/gchaincl/httplab.git')
 md5sums=(SKIP)
 
+prepare() {
+	cd "$pkgname"
+	GOPATH="$srcdir"
+	go get -v -d
+}
+
 build() {
 	cd "$pkgname"
 	go build
 }
 
 package() {
 	cd "$pkgname"
-	install -D -s -m644 httplab "${pkgdir}/usr/bin/httplab"
+	install -Dsm755 httplab "${pkgdir}/usr/bin/httplab"
```

Comment:
I only have the community version of go installed - which is at 1.8.  Not a golang dev and the only other package I use that has dependecies on go is git-lfs (which is what I used as a reference for GOPATH to get httplab to install locally with makepkg)

Attaching screenshot of the failed build if I try to install from AUR:
![2017-03-01-144414_1336x1638_scrot](https://cloud.githubusercontent.com/assets/597310/23477933/9e4253ca-fe8d-11e6-948d-8de7810e198a.png)